web/src/routes/OrganizationsList/Organization/Tabs/TeamsAndMembership/TeamsView/SetRepoPermissionsModal/SetRepoPermissionForTeamModal.tsx

```diff
@@ -217,7 +217,8 @@ export default function SetRepoPermissionForTeamModal(
                       rowIndex,
                       onSelect: (_event, isSelecting) =>
                         onSelectRepoPerm(repoPerm, rowIndex, isSelecting),
-                      isSelected: isItemSelected(repoPerm),
+                      isSelected:
+                        isItemSelected(repoPerm) || repoPerm.role !== 'none',
```

Comment:
fyi - this fixes an issue I uncovered where the checkbox wasn't being selected in team's view list (for set repository perm modal)